### PR TITLE
Add possibility to also package symbols within ipa

### PR DESCRIPTION
**Pull request by BastianKusserow**
If you like to also package symbols within your ipa, you can do this now via

```gradle
packaging {
  packageSymbols = true
}
```
This should resolve https://github.com/openbakery/gradle-xcodePlugin/issues/454

I am not sure if target should be develop or main, feel free to change

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,7 @@
 allprojects {
 	apply plugin: 'java-gradle-plugin'
 	apply plugin: 'groovy'
+	apply plugin: "maven"
```
Comment (BastianKusserow) [inline build.gradle:4]: I've reapplied the plugin, since `gradle install` is available with this, so the plugin can be tested locally

**Comment by renep** [comment]
I'm wondering if the `packageSymbols=true`should be default and with `false` it can be disabled.

**Comment by BastianKusserow** [comment]
wasn't sure if this is too much of a breaking change, but I definitely support enabling this by default!

**Comment by renep** [comment]
Then make is so 😉

**Comment by BastianKusserow** [comment]
sure, done :)

**Comment by BastianKusserow** [comment]
Hey @renep, is there a chance we could merge that soon'ish? :)

**Pull request merged**